.github/workflows/release.yml

```diff
@@ -0,0 +1,53 @@
+name: Release Artifacts for AI examples
+on:
+  workflow_dispatch:
+    inputs:
+      app:
+        type: choice
+        description: Which app to build and push
+        options: 
+        - newsfeeder-ts
+      tag:
+        type: string
+        description: 'new tag for this push'
+        required: true
+
+env:
+  REGISTRY: ghcr.io
+
+jobs:
+  echo-inputs:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Echo Inputs
+        run: |
+          echo app: ${{ github.event.inputs.app }}
+          echo tag: ${{ github.event.inputs.tag }}
+
+  build-and-push:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
```

Comment:
As I understand it, this workflow pulls the latest changes and pushes it to ghcr.io with the specified tag.

Wouldn't we want this workflow to pull the tag we specified as input? e.g. if I ask this workflow to build `newsfeeder-ts:v1.0.0`, wouldn't I want it to pull v1.0.0?

Would it be better if we omitted the tag as a parameter and always push it as "latest"?